icu4c/source/common/caniter.cpp

```diff
@@ -208,10 +208,10 @@ void CanonicalIterator::setSource(const UnicodeString &newSource, UErrorCode &st
         goto CleanPartialInitialization;
     }
 
-    // i should initialy be the number of code units at the 
```

Comment:
I think this should probably stay as `i` since that is the variable name.

Reply:
Ahhh that makes sense haha